scripts/doc-snippets.ts

```diff
@@ -47,7 +47,7 @@ function reindented(spaces: number, lines: string[]): string[] {
 const mkpath = require("mkpath");
 
 const sdkDir = path.resolve(__dirname, "..");
-const outDir = path.resolve(sdkDir, "dist/doc");
+const outDir = path.resolve(sdkDir, "dist/snippets");
```

Comment:
snippets is quite generic - `doc-snippets`?

Reply:
Will do!